macro-test/src/test/scala/scala/pickling/generator/scalasymbols/TestScalaSymbols.scala

```diff
@@ -54,34 +54,34 @@ class TestScalaSymbols extends FunSuite {
 
   test("fieldTypesWithTypeArgs") {
     assert(Compat.varTypes[ParentTag.Foo[String, Int]].toSet == Set(
-      "scala.Tuple2[java.lang.String,scala.Int]", "scala.Int"
+      "(String, Int)", "Int"
```

Comment:
I feel like the above representation `scala.Tuple2[java.lang.String,scala.Int]` is more accurate than `(String, Int)`. Is this happening because you're now using `toString` of the type implementation? I'm not sure if this human-friendly display name should be taken seriously.